Gems/Multiplayer/Code/Source/NetworkEntity/EntityReplication/EntityReplicationManager.cpp

```diff
@@ -559,9 +559,10 @@ namespace Multiplayer
         {
             if (entityReplicator->IsMarkedForRemoval())
             {
+                const AZ::Entity* entity = entityReplicator->GetEntityHandle().GetEntity();
```

Comment:
You might want a maybe_unused on this, since if AZLOG_WARN is ever ifdef'd out by a customer you'd get a warning on entity being an unused variable..

Reply:
+1. Also would cause problems for release builds i think.